front/src/applications/stdcm/utils/formatStdcmIntoSpaceTimeData.ts

```diff
@@ -1,24 +1,23 @@
-import type { ProjectPathTrainResult } from 'common/api/osrdEditoastApi';
+import type { TrainSpaceTimeData } from 'applications/operationalStudies/types';
```

Comment:
Is it ok that stdcm depends on a type from operational studies ? We also have to deal with `signalUpdates` even though we don't need them

Reply:
No I think that the type TrainSpaceTimeData should be in module/SpaceTimeChart :/
As it was already there, I did not move it. Do you want me to move it now ? I can also do it in another PR.

Concerning the signalUpdates, I think we should always keep it in the TrainSpaceTimeData type, as we might use it every time we are displaying the trains on the SpaceTimeChart

Reply:
Yep, it can be done in a different PR :)